_posts/2020-11-25-cockpit-233.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Cockpit 233
+author: lis
+date: 2020-11-25
+tags: cockpit linux
+slug: cockpit-233
+category: release
+summary: Cockpit with various bug fixes and performance improvements
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly. Here are the release notes from Cockpit version 233.
+
+### Non-admin users no longer see Cockpit in motd
+
+The motd message suggesting how Cockpit can be enabled or used is now
+shown only to admin users.  This depends on PAM 1.5.0 (which is not yet
+released) but the needed patches have been backported to Fedora 33 and
+may land in other distributions in the future.
```

Comment:
```suggestion
### MOTD is not displayed to non-admin users

The `/etc/motd` message suggesting how Cockpit can be enabled or used is now
shown only to admin users.  This depends on PAM 1.5.0 (which is not yet
released) but the needed patches have been backported to Fedora 33 and
may land in other distributions in the future.

*[MOTD]: Message of the day
```

(I think I did [the abbreviation footnote thing](https://kramdown.gettalong.org/syntax.html#abbreviations) properly?)

Reply:
thanks for doing this.  i've taken it verbatim.

note that pitti had marked this as release-note, so that's why I included it

Reply:
I disagree with the motd change, actually.  Some if what you wrote is technically incorrect (ie: this is not `/etc/motd`, nor is it *the* "MOTD")  and I also prefer the headings of release notes to make it clear what the change is rather than stating a simple fact (as your suggested heading does).